frontend/src/components/projects/ProjectsAssociatedSamples.tsx

```diff
@@ -64,14 +139,21 @@ export const ProjectsAssociatedSamples = ({ projectID: currentProjectID } : Proj
         SAMPLE_COLUMN_DEFINITIONS.VOLUME,
         SAMPLE_COLUMN_DEFINITIONS.CREATION_DATE,
         SAMPLE_COLUMN_DEFINITIONS.DEPLETED,
+        {
+            columnID: 'STUDY-STEP',
+            title: 'Study - Step',
```

Comment:
"Study step" would be better this is the current study step being displayed. No need to hyphenation i think.

Reply:
I have feeling that would look worse if a sample is assigned to multiple studies, considering that the column is just a comma-separated set of study-step.

Reply:
Ok ... it looks weird any way for multiple steps ... the list format is not totally satisfying but i am not sure how we would replace that.